specifyweb/frontend/js_src/lib/components/FormParse/cells.ts

```diff
@@ -329,6 +332,9 @@ export function parseFormCell(
       : cellType === 'Label'
       ? 'right'
       : 'left',
+    verticalAlign: f.includes(cellVerticalAlign, verticalAlign)
+      ? verticalAlign
+      : 'stretch',
```

Comment:
So it looks like, before this PR, the vertical alignment was center.
But after this PR, it would be top instead.
Thus, you should either:
1. Test the implications of this change in a variety of scenarious
2. Do what line 332 in this file does, and apply 'stretch' only for SubView cells, and center for all others